apps/financial-aid/web-veita/src/components/Profile/ApplicationProfile.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useMemo, useContext } from 'react'
 import { Box } from '@island.is/island-ui/core'
 
-import * as styles from './ApplicationProfile.css'
+import * as styles from './Profile.css'
```

Comment:
Im using the same styles for ApplicationProfile and Employyprofile, and they are under a folder called profile, thought it would be nice to use together 👬